src/SurveyDesign.jl

```diff
@@ -200,7 +197,7 @@ end
 """
     StratifiedSample <: AbstractSurveyDesign
 
-Survey design sampled by stratification.
+A stratified sample dataset can be loaded from a data frame into a `StatifiedSample` object for downstream analyses. 
```

Comment:
```suggestion
Survey design sampled by stratification.
```

Reply:
We can do “Stratified sample survey design” if we provide suitable information above the docstrings of `AbstractSurveyDesign`

Reply:
What suitable information?